### DISCUSSION TITLE
tab height is too small/hard to read the text on the tab

### DISCUSSION THREAD

**Discussion by qlldominic** [sumatrapdfreader/sumatrapdf#3735] [Q&A]
I guess some people has asked similar question. I am glad the newer version has introduce the function of change the width of the tab; 
however, the current tabs are really hard to read.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/146145147/79250c6b-a030-4051-ac98-cf7c5b9f8771)

Can we get bigger text in the tab (vertical), something like the adobe:
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/146145147/e44043ba-0b2b-4437-9e7d-af2ef8581ce1)

**Comment by GitHubRulesOK** [comment]
It depends how you measure small here its about same height but SumatraPDF top bar need to be dual action without all the other border data.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/86f7b38d-2322-478e-893c-3cf8698d7ef9)

**Comment by FedirVolkov** [comment]
Bump! It's very inconvenient on touchscreen device with such small tab height! Also X button (to close tab) is not well responsive for touch, I have to tap it multiple times to close the tab.

**Reply by GitHubRulesOK** [reply]
Current Pre-release version vas variable settings see https://github.com/sumatrapdfreader/sumatrapdf/issues/5136#issuecomment-3408446152 but each modified value affects others so adjust with care.

**Reply by FedirVolkov** [reply]
Version 3.6.17065. I found that changing UIFontSize changing size of tabs too, but menus and fonts are becoming too big very fast, not so convinient. Also because font is huge I still can see just a few first letters of the name of tabs (when there is a lot of them).  Also I tried setting CustomScreenDPI but that doesn't change anything.

**Reply by ghost** [reply]
I also agree, and I am facing a similar problem. Perhaps you could consider making the hitbox of the X button larger, as shown in photo 2. I am not sure why, but on Sumatra I often miss the close button.
<img width="310" height="57" alt="1" src="https://github.com/user-attachments/assets/465057db-01a7-402f-8ce1-cd3f178a5a92" />
<img width="310" height="57" alt="2" src="https://github.com/user-attachments/assets/44a9ae0e-f853-4b83-9eda-b58be546016f" />

**Comment by kjk** [comment]
I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/5220 to track this
